src/main/java/frc/robot/commands/shooter/AdvancedSpinningShot.java

```diff
@@ -54,10 +53,6 @@ public void execute() {
                 shooter.setShooterMotorLeftRPM(shooterSpeed.getLeftMotorSpeed());
             }
         }
```

Comment:
No light strip color on Advanced Spinning Shot?

Reply:
OK so this is only used on the new joyLeftButton3.

Reply:
would override auto ramp desiredPos indication